logbook-json/src/main/java/org/zalando/logbook/json/JsonPathBodyFilters.java

```diff
@@ -0,0 +1,107 @@
+package org.zalando.logbook.json;
+
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import lombok.RequiredArgsConstructor;
+import net.minidev.json.JSONArray;
+import org.zalando.logbook.BodyFilter;
+
+import javax.annotation.Nullable;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class JsonPathBodyFilters {
+
+    @RequiredArgsConstructor
+    public static class JsonPathBodyFilterBuilder {
+        private final JsonPath jsonPath;
+
+        public BodyFilter delete() {
+            return (JsonBodyFilter) documentContext -> documentContext.delete(jsonPath).jsonString();
+        }
+
+        public BodyFilter replace(String replacement) {
+            return replace(replacement, false, String.class);
+        }
+
+        public BodyFilter forceReplace(String replacement) {
```

Comment:
What does force do?

Reply:
It forces replace :))
For example, we have json `{"test":{}}` and we call `jsonPath("$.test").replace("XXX")`. In such case nothing will be changed, because $.test is neither json array nor primitive. But, if we call `jsonPath("$.test").forceReplace("XXX")`, we'll get `{"test":"XXX"}` as the result of filtering.

Reply:
I was expecting that the *force* behavior is the default. I wouldn't have any special semantics for strings, arrays or objects. It should always behave the same way. Just replace every matching node with the given string.